std_msgs/Header header
ThrotCmd throt_cmd_msg
BrakeCmd brake_cmd_msg
ParkCmd park_cmd_msg
SteerCmd steer_cmd_msg
BodyCmd body_cmd_msg
PowerCmd power_cmd_msg
CloudDriveCmd cloud_drive_cmd_msg
CloudBodyCmd cloud_body_cmd_msg
VehVinWrite veh_vin_write_msg
#imu
ImuInputCmd imu_input_cmd_msg